<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="amountEdit">
       <property name="placeholderText">
        <string>金额</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="date">
        <date>
         <year>2025</year>
         <month>5</month>
         <day>22</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="noteEdit">
       <property name="placeholderText">
        <string>备注</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="incomeButton">
       <property name="text">
        <string>收入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expenseButton">
       <property name="text">
        <string>支出</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QDateEdit" name="startDateEdit">
         <property name="date">
          <date>
           <year>2025</year>
           <month>5</month>
           <day>22</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>至</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endDateEdit">
         <property name="date">
          <date>
           <year>2025</year>
           <month>5</month>
           <day>22</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterButton">
         <property name="text">
          <string>筛选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearFilterButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>清除筛选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="recordTable">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>600</height>
          </size>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <column>
          <property name="text">
           <string>序号</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <bold>false</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>金额</string>
          </property>
         </column>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>总收入：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalIncomeLabel">
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>总支出：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalExpenseLabel">
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>结余：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="balanceLabel">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>保存数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>加载数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>导出数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="categoryStatsButton">
           <property name="text">
            <string>分类统计</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="compressButton">
           <property name="text">
            <string>压缩数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="restoreButton">
           <property name="text">
            <string>恢复备份</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
